apiserver/facades/controller/caasapplicationprovisioner/provisioner_test.go

```diff
@@ -97,7 +97,7 @@ func (s *CAASApplicationProvisionerSuite) setupAPI(c *gc.C) *gomock.Controller {
 	s.applicationService = NewMockApplicationService(ctrl)
 	s.leadershipRevoker = NewMockRevoker(ctrl)
 	s.resourceOpener = NewMockOpener(ctrl)
-	newResourceOpener := func(appName string) (jujuresource.Opener, error) {
+	newResourceOpener := func(_ context.Context, appName string) (jujuresource.Opener, error) {
```

Comment:
name or don't name things, don't do both with an underscore :p 
```suggestion
	newResourceOpener := func(context.Context, string) (jujuresource.Opener, error) {
```